3-networks/modules/vpn-ha/main.tf

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/******************************************
+  HA VPN configuration
+ *****************************************/
+
+locals {
+  vpc_name     = "${var.environment_code}-shared-${var.vpc_label}"
+  network_name = "vpc-${local.vpc_name}"
```

Comment:
Both `vpc_name`  and `network_name` could be variables since they are defined in the context the module is called.